migrator/migrations/n_52_to_n_53_postgres_simple_access_scopes/migration.go

```diff
@@ -251,6 +255,52 @@ func migrateRoles(gormDB *gorm.DB, postgresDB *pgxpool.Pool, legacyStore legacyr
 	return nil
 }
 
+func getReportConfigurationScopeID(reportConfiguration *storage.ReportConfiguration) (string, error) {
+	reportConfigurationScopeID := strings.TrimPrefix(reportConfiguration.GetScopeId(), accessScopeIDPrefix)
+	if replacement, found := accessScopeIDMapping[reportConfigurationScopeID]; found {
+		reportConfigurationScopeID = replacement
+	}
+	_, accessIDParseErr := uuid.FromString(reportConfigurationScopeID)
+	if accessIDParseErr != nil {
+		log.WriteToStderrf("failed to convert report configuration to postgres format, bad scope ID. Role [%s], error %v", reportConfiguration.GetName(), accessIDParseErr)
```

Comment:
+ would be nice to specifically print the `reportConfigurationScopeID` as well.
```suggestion
		log.WriteToStderrf("failed to convert report configuration to postgres format, bad scope ID. Report [%s], error %v", reportConfiguration.GetName(), accessIDParseErr)
```

Reply:
Good idea. Now both ID and name are reported in the log.